app/src/openxr/cpp/OpenXRGestureManager.cpp

```diff
@@ -99,8 +99,12 @@ OpenXRGestureManagerFBHandTrackingAim::aimPose(const XrTime predictedDisplayTime
 
 bool
 OpenXRGestureManagerFBHandTrackingAim::systemGestureDetected(const vrb::Matrix& palm, const vrb::Matrix& head) const {
-#ifdef PICOXR
-    // Pico does not correctly implement the SYSTEM_GESTURE_BIT_FB flags.
+#if defined(PICOXR) || defined(LYNX)
+    // Pico and Lynx do not correctly implement the SYSTEM_GESTURE_BIT_FB flags.
+#ifdef LYNX
+    // Lynx does not correctly implement the XR_HAND_TRACKING_AIM_VALID_BIT_FB flags (it's always active)
+    return palmFacesHead(palm, head);
+#endif
```

Comment:
Maybe we can simplify this a bit and do
```
if LYNX
elif PICOXR
else
```
?

Reply:
Indeed ! Thanks.